CONTRIBUTING.md

```diff
@@ -189,9 +189,15 @@ in the body of the pull request to indicate tasks are pending.
 
 At this stage, one should expect comments and review from other contributors. You
 can add more commits to your pull request by committing them locally and pushing
-to your fork until you have satisfied all feedback.
+to your fork.
 
-Note: Code review is a burdensome but important part of the development process, and as such, certain types of pull requests are rejected. In general, if the **improvements** do not warrant the **review effort** required, the PR has a high chance of being rejected. It is up to the PR author to convince the reviewers that the changes warrant the review effort, and if reviewers are "Concept NACK'ing" the PR, the author may need to present arguments and/or do research backing their suggested changes.
+As the pull request author you are expected to reply to any review comments
```

Comment:
Here it is "as the pull request author **you** are". But later it moves from "you" back to the PR "author may need to..."

Try to be consistent? Either it is all "you should" or "the PR author should". I think I prefer the latter here. It gets confusing if it is not clear whether "you" is the PR author or the PR reviewer.

Reply:
This section uses "you", the others do not, so I stuck to that.